clusters/ztp-siteconfig/puffin.wpc.test/puffin-siteconfig.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   baseDomain: "wpc.test"
   pullSecretRef:
     name: "assisted-deployment-pull-secret"
-  clusterImageSetNameRef: "img4.13.4-x86-64-appsub"
+  clusterImageSetNameRef: "openshift-4.14.0-ec.4"
```

Comment:
I think this change can't be applied on a running cluster, so we should remove the SiteConfig and push it back to redeploy